src/main/java/reborncore/mixin/common/MixinWorld.java

```diff
@@ -39,7 +39,7 @@
 @Mixin(World.class)
 public class MixinWorld {
 
-	@Shadow
+	/* FIXME @Shadow
```

Comment:
There is a fabric event for this now. Should prob move over to it in the 1.16 branch and then pull that change in here.

Reply:
Rebased on 1.16